restorer_test.go

```diff
@@ -84,7 +89,7 @@ func testRestorer(t *testing.T, when spec.G, it spec.S) {
 			})
 			when("there is a cache=false layer", func() {
 				it.Before(func() {
-					meta := "[types]\n  cache=false"
+					meta := ""
```

Comment:
This is confusing to me.  The when states there will be a `cache=false` layer but then I don't see anything setting that up.

Reply:
This setup (having two `when`s for `cache=true` and `cache=false`) makes more sense when layer types information is available in `<layer>.toml`. From the restorer's perspective, when there is no cache, all layers are `cache=false` layers (when bp api is 0.6+). We're always using `latestBuildpackAPI` for these tests, so I agree it's a bit confusing.

Since we are moving toward using table tests in our units it might make sense to keep the current setup and MAYBE add some permutations where we vary the buildpack api as well (as opposed to just the platform api). WDYT?

Reply:
Hmm ok. Yeah I guess for regression safety we should probably introduce support BP API tests for at some paths like this.

Reply:
Sounds good. Would you be okay deferring that to the feature work in 0.12.0?

Reply:
I am fine with that as long as we are confident we didn't break `< 0.6` BP API behavior, since that is the majority of known buildpacks.

Reply:
This is true, I took a stab at turning the restorer units into table tests. The diff is now horrible, might be more readable in an IDE.